.github/workflows/kind-e2e.yaml

```
@@ -35,18 +39,46 @@ jobs:
       run: |
         make toolchain
         make install-kwok
-        KWOK_REPO=kind.local KIND_CLUSTER_NAME=chart-testing make apply-with-kind
+        export KWOK_REPO=kind.local
+        export KIND_CLUSTER_NAME=chart-testing
+        make apply-with-kind
     - name: ping cluster
       shell: bash
       run: | 
+        sleep 15
         kubectl get pods -n kube-system | grep karpenter 
         kubectl get nodepools
         kubectl get pods -A
         kubectl describe nodes
+    - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+      with:
+        repository: nathangeology/karpenter_evaluate
+        path: ./karpenter_eval/ # Installs to a folder in the Karpenter repo for the test
+        ref: "3f4cebb703bd136f5034c8b5bf3e6c32e97d1ae6"
+        fetch-depth: 0
+    - name: install KPI report dependencies
+      shell: bash
+      run: |
+        pip install awswrangler
+        pip install tabulate
+        pip install prometheus-api-client
+        pip install ./karpenter_eval/
     - name: run test suites
       shell: bash
-      run: | 
+      run: |
+        OUTPUT_DIR=$(mktemp -d) 
+        export OUTPUT_DIR
```

You should be able to combine this into one 
```suggestion
        export OUTPUT_DIR=$(mktemp -d) 
```

---

Making this change resulted in a shellcheck error: .github/workflows/kind-e2e.yaml:69:7: shellcheck reported issue in this script: SC2155:warning:1:8: Declare and assign separately to avoid masking return values [shellcheck]